pages/index.tsx

```diff
@@ -1,15 +1,50 @@
-import Header from '@/components/Header';
+import { heyTheme } from '@/public/theme/theme';
+import { Button } from '@chakra-ui/react';
+import styled from '@emotion/styled';
 import Head from 'next/head';
+import Image from 'next/image';
+import { RiKakaoTalkFill } from 'react-icons/ri';
 
 export default function Home() {
+  const { sub, lightOrange } = heyTheme.colors.hey;
+
   return (
     <>
-      <Header />
       <Head>
-        <title>next-ts-boilerplate</title>
+        <title>Hey, cake</title>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
       </Head>
-      <main>next-ts-boilerplate</main>
+      <ImageContainer>
+        <Image src="/images/logo.png" alt="로고" width={300} height={300} />
+        <ImageTitle>커스텀 케이크 제작 의뢰 플랫폼</ImageTitle>
+      </ImageContainer>
+      <ButtonContainer>
```

Comment:
```suggestion
      <LoginButtonContainer>
```

변수 이름을 좀 더 구체적으로 하면 어떨까요? 이렇게요! `LoginButtonContainer`